extensions/cdc-debezium/src/main/java/com/hazelcast/jet/cdc/impl/CdcSourceP.java

```diff
@@ -413,11 +413,18 @@ private static void log(ILogger logger, String name, String property, Object val
     }
 
     private static long extractTimestamp(SourceRecord record) {
-        if (record.valueSchema().field("ts_ms") == null) {
+        boolean noValueTsMs = record.valueSchema().field("ts_ms") == null;
```

Comment:
Could we extract `ts_ms` to more meaningful constant?

Reply:
f7f35da7c440a1b20f1224aa0474456e97ddf7ae